src/modules/java.rs

```diff
@@ -0,0 +1,71 @@
+use std::process::Command;
+
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Java version
+///
+/// Will display the Java version if any of the following criteria are met:
+///     - Current directory contains a file with a `.java` extension
+///     - Current directory contains a `pom.xml` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_java_project = context
+        .new_scan_dir()
+        .set_files(&["pom.xml"])
+        .set_extensions(&["java"])
+        .scan();
+
+    if !is_java_project {
+        return None;
+    }
+
+    match get_java_version() {
+        Some(java_version) => {
+            const JAVA_CHAR: &str = "☕ ";
+            let module_color = Color::RGB(166, 42, 42).bold();
+
+            let mut module = context.new_module("java")?;
+            module.set_style(module_color);
```

Comment:
Up until about 5 minutes ago, this was the correct way to set the color in modules. Unfortunately, you happened to open this PR right on the back of a pretty big change in how styling is handled in starship.

Fortunately, the change won't be too bad for this module. Take a look at the new e.g. [rust.rs](https://github.com/starship/starship/blob/master/src/modules/rust.rs#L27) for the new syntax: essentially, you ask the module to get its style, and if that fails, fall back to your selected color.